ios/RNTPTDocumentViewManager.m

```diff
@@ -564,6 +564,17 @@ - (void)selectAnnotationForDocumentViewTag:(NSNumber *)tag annotationId:(NSStrin
     }
 }
 
+- (NSString *)getPageCropBoxForDocumentViewTag:(NSNumber *)tag pageNumber:(NSNumber *)pageNumber
```

Comment:
For `pageNumber`, should use a `NSInteger` (which is a typedef to a basic type) rather than an `NSNumber` object, which is consistent with the method `selectAnnotationForDocumentViewTag`.